AutomatedTesting/ShaderVariants/Shaders/ScreenSpace/MobileDeferredFog.shadervariantlist

```diff
@@ -0,0 +1,25 @@
+{
```

Comment:
If you used SMC to create this, can you add the steps as a comment at the top for someone who wants to extend it in future. And lets move this to the core engine.